### PR TITLE
Update apps-supported-intune-apps.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Microsoft lens to the document

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/2770

File: memdocs/intune/apps/apps-supported-intune-apps.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ The below apps support the core Intune App Protection Policy settings.
 |Microsoft Dynamics CRM|[iOS](https://apps.apple.com/app/microsoft-dynamics-crm/id678800460)|✔|No settings|
 |Microsoft Kaizala|[Android](https://play.google.com/store/apps/details?id=com.microsoft.mobile.polymer)|✔|No settings|
 |Microsoft Kaizala|[iOS](https://apps.apple.com/app/microsoft-kaizala/id1112208399)|✔|No settings|
+|Microsoft Lens|[Android](https://play.google.com/store/apps/details?id=com.microsoft.office.officelens)|✔|No settings|
+|Microsoft Lens|[iOS](https://apps.apple.com/us/app/microsoft-lens-pdf-scanner/id975925059|✔|No settings|
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-supported-intune-apps.md:114]:
```suggestion
|Microsoft Lens|[iOS](https://apps.apple.com/app/microsoft-lens-pdf-scanner/id975925059)|✔|No settings|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@msbemba Please implement the suggested changes. thanks

**Comment by msbemba** [comment]
@yogkumgit  There are no pending changes to be committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder

**Pull request closed without merging**